connectors/uipath/element-templates/uipath-prototype-add-queue-item.json

```diff
@@ -0,0 +1,392 @@
+      }
+    },
+    {
+      "id": "method",
+      "label": "Method",
+      "group": "endpoint",
+      "type": "Dropdown",
+      "value": "post",
+      "choices": [
+        {
+          "name": "GET",
+          "value": "get"
+        },
+        {
+          "name": "POST",
+          "value": "post"
+        },
+        {
+          "name": "PATCH",
+          "value": "patch"
+        },
+        {
+          "name": "PUT",
+          "value": "put"
+        },
+        {
+          "name": "DELETE",
+          "value": "delete"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "method"
+      }
+    },
+    {
+      "label": "URL",
+      "description": "Request URL for adding items to the queue. Please, replace '{{cloudUrl}}', '{{cloudOrg}}' and '{{cloudTenant}}' with the the values from your uipath orchestrator.",
+      "group": "endpoint",
+      "type": "String",
+      "feel": "required",
+      "value": "=\"https://{{cloudUrl}}/{{cloudOrg}}/{{cloudTenant}}/odata/Queues/UiPathODataSvc.AddQueueItem",
+      "editable": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "url"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "pattern": {
+          "value": "^(=|https?://).*",
+          "message": "Must be a http(s) URL."
+        }
+      }
+    },
+    {
+      "label": "Headers",
+      "description": "Map of HTTP headers to add to the request. Please, replace '{{organizationUnitId}}' with the value from your uipath orchestrator.",
+      "group": "endpoint",
+      "type": "String",
+      "feel": "required",
+      "value": "={\"X-UIPATH-OrganizationUnitId\":\n\"{{organizationUnitId}}\", \n\"Content-Type\": \"application/json\"}",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "headers"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Request Body",
+      "description": "JSON payload to send with the request. Please, replace values with curls (e.g. '{{deferDate}}') with real values ('2022-12-28'). Hints: queueName is the only mandatory field; specificContent should have json format; priority should have one of the values: 'Low', 'Normal' or 'High' ",
+      "group": "input",
+      "type": "String",
+      "feel": "required",
+      "value": "={\n\t\"itemData\": {\n\t\t\"DeferDate\":\"{{deferDate}}\", \n\t\t\"DueDate\":\"{{dueDate}}\",\n\t\t\"Priority\":\"{{priority}}\",\n\t\t\"Name\":\"{{queueName0}}\",\n\t\t\"SpecificContent\":\"{{specificContent}}\",\n\t\t\"Reference\":\"{{reference}}\" \n\t}\n}",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "method",
+        "oneOf": [
+          "post",
+          "put",
+          "patch",
+          "delete"
+        ]
+      }
+    },
+    {
+      "label": "Bearer Token",
```

Comment:
hide it, if is not needed.

Reply:
We decided to keep it as it is a valid authentication method for uipath.